skribenten-backend/build.gradle.kts

```diff
@@ -95,4 +95,6 @@ dependencies {
     testImplementation("io.ktor:ktor-client-mock:$ktorVersion")
     testImplementation("io.mockk:mockk:${mockkVersion}")
     testImplementation("com.natpryce:hamkrest:$hamkrestVersion")
+    testImplementation("org.assertj:assertj-core:3.25.3")
```

Comment:
Legg merke til at vi allerede har hamkrest, som tilbyr tilsvarende funksjonalitet. Helt greit for meg å evt. bytte til assertj-core om du liker det bedre, men vi trenger muligens ikke begge?

Reply:
Bytter :) Liker assertj bedre.